queries.sql

```diff
@@ -116,4 +116,71 @@ SELECT owners.full_name, COUNT(animals.id) AS num_animals
 FROM owners
 LEFT JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.id
-ORDER BY num_animals DESC;
\ No newline at end of file
+ORDER BY num_animals DESC;
+
+-- Who was the last animal seen by William Tatcher?
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230588289-bbf31498-5415-453f-af46-d1c9f932e7e8.png)

- [ ] It would be great if you remove the highlighted things in the image above ⬆️ to make your readme file descriptive and clean because these are just instructions for the first time when you make your readme file 🤔 After making it you are supposed to remove all the instructions and unnecessary things 👍